### PR TITLE
Modify DataServlet to contain ArrayList with hard-coded messages

### PR DESCRIPTION
**Pull request by johnjara**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,39 @@
...
+    
+    // Send the JSON as the response
+    String json = convertToJson(comments);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+    // function to convert comments to JSON
+    private String convertToJson(ArrayList comments) {
+    String json = "{";
```
Comment (gitmnk) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:46]: indentation here so we know these lines are part of the method convertToJson

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,39 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```
Comment (qidichen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:22]: Generally it's not a good practice to use import *. Please specify which sub-library you are using such as ArrayList.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,39 @@
...
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello John!");
+    
+    // make arraylist with three hard-coded values
+   ArrayList<String> comments = new ArrayList<String>(); // Creating arraylist
```
Comment (qidichen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:33]:
```suggestion
   List<String> comments = new ArrayList<String>(); // Creating arraylist 
```

Better to use interface for holding the result.

Comment (johnjara) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I received this error when switching ArrayList<String> to List<String>: 

[ERROR] /home/jjara/software-product-sprint/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:[
45,34] cannot find symbol
  symbol:   class List

Comment (gitmnk) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: You need to import it explicitly now that you have changed the import to just for Arraylist

**Pull request merged**